sourcegraph/sourcegraph.proto

```diff
@@ -142,7 +146,9 @@ message Repo {
 	// Fork is whether this repository is a fork.
 	bool fork = 12;
 
-	// Mirror is whether this repository is a mirror.
+	// Mirror indicates whether this repo's canonical location is on
+	// another server. Mirror repos track their upstream and are not
+	// eligible for discovery on this server.
 	bool mirror = 13;
```

Comment:
Ok, never mind. I've more carefully reviewed what you've wrote and the comments and now I understand. `mirror`ed repos are repos that are kept in sync with their origin (or upstream).